aslam_cv_pipeline/src/visual-npipeline.cc

```diff
@@ -70,6 +70,21 @@ bool VisualNPipeline::processImageBlockingIfFull(size_t camera_index, const cv::
   return false;
 }
 
+bool VisualNPipeline::processImageNonBlockingDroppingIfFull(
```

Comment:
is this name misleading because it could indicate that actually this image is dropped if full, instead of the oldest complete nframe? if yes, consider renaming to something like ...DroppingOldestCompleteNFrameIfFull?